autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/ActivityStatus.java

```diff
@@ -5,6 +5,12 @@ public enum ActivityStatus {
     METRICS_COLLECTION_SUCCESS,
     METRICS_COLLECTION_FAILED,
 
+    MANDATORY_UPSCALE,
```

Comment:
I added these states to differentiate between an UPSCALE/DOWNSCALE being triggered and readjusting the cluster to conform to the limits defined in the autoscale policy. These states will be populated when the policy is created/edited and the cluster requires an explicit upscale/downscale so that upper and lower bounds match that of the policy. Additionally, similar applicable states here could be `UPSCALE/DOWNSCALE_TRIGGERED`, however, I feel we must populate those states only after the corresponding CB invocation is made. Hence, I added separate states for these "mandatory adjustments"